storage/client_split_test.go

```diff
@@ -85,13 +85,13 @@ func TestStoreRangeSplitAtIllegalKeys(t *testing.T) {
 }
 
 // TestStoreRangeSplitAtTablePrefix verifies a range can be split at
-// TableDataMin and still gossip the SystemConfig properly.
+// UserTableDataMin and still gossip the SystemConfig properly.
 func TestStoreRangeSplitAtTablePrefix(t *testing.T) {
 	defer leaktest.AfterTest(t)
 	store, stopper := createTestStore(t)
 	defer stopper.Stop()
 
-	key := keys.TableDataMin
+	key := keys.UserTableDataMin
```

Comment:
I'm not quite sure why this was working before. `keys.TableDataMin` lies in the system DB span and is not a valid split key.